src/v/pandaproxy/schema_registry/configuration.cc

```diff
@@ -42,6 +42,13 @@ configuration::configuration()
       "api_doc_dir",
       "API doc directory",
       {},
-      "/usr/share/redpanda/proxy-api-doc") {}
+      "/usr/share/redpanda/proxy-api-doc")
+  , rest_require_auth(
```

Comment:
I'd prefer if this was spelled out as authentication to avoid doubt as to whether it's authorization.